I have an little problem with update from 2.18 to 2.19

I have an little problem with update from 2.18 to 2.19 
Also with angular from 20 to 21

This code worked on 2.18 + angular 20
 this.mapInstance?.pm.Toolbar.copyDrawControl('drawMarker',{
                name: 'hospitalMarker',title: 'Hospital',
                block: 'custom', className: 'pt_hospital'
    }).drawInstance.setOptions({markerStyle: { icon: HospitalIcon },});

But with 2.19 + angular 21 it give me 2 build errors:
Application bundle generation failed. [25.354 seconds] - 2026-01-13T07:41:35.664Z

✘ [ERROR] TS2349: This expression is not callable.
  Not all constituents of type 'DrawShape | ((...args: any[]) => any)' are callable.
    Type 'DrawShape' has no call signatures. [plugin angular-compiler]

    src/app/page05/page05.ts:69:20:
      69 │     }).drawInstance.setOptions({markerStyle: { icon: HospitalIcon ...
         ╵                     ~~~~~~~~~~

✘ [ERROR] TS4111: Property 'setOptions' comes from an index signature, so it must be accessed with ['setOptions']. [plugin angular-compiler]

    src/app/page05/page05.ts:69:20:
      69 │     }).drawInstance.setOptions({markerStyle: { icon: HospitalIcon ...
         ╵                     ~~~~~~~~~~

Please help me.
Best regards

--- a/leaflet-geoman.d.ts
+++ b/leaflet-geoman.d.ts
@@ -1084,9 +1084,9 @@ declare module 'leaflet' {
       /** Creates a copy of a draw Control. Returns the drawInstance and the control. */
       copyDrawControl(
         copyInstance: string,
-        options: CustomControlOptions
+        options: CustomControlOptions | string
       ): {
-        drawInstance: Draw;
+        drawInstance: DrawShape;
         control: L.Control;
       };
 
@@ -1632,6 +1632,10 @@ declare module 'leaflet' {
       /** Returns the active shape. */
       getActiveShape(): SUPPORTED_SHAPES;
 
+      /** Creates a new draw instance for custom controls. */
+      createNewDrawInstance(name: string, jsClass: string): DrawShape;
+
+      /** Access shape-specific draw instances (e.g., Draw.Line, Draw.Marker) */
       [key: string]: DrawShape | ((...args: any[]) => any);
     }
 
@@ -1647,6 +1651,15 @@ declare module 'leaflet' {
 
       /** Get options */
       getOptions(): DrawModeOptions;
+
+      /** The current options for this draw shape */
+      options?: DrawModeOptions;
+
+      /** The toolbar button name associated with this draw instance */
+      toolbarButtonName?: string;
+
+      /** The shape identifier */
+      _shape?: string;
     }
 
     interface LassoDraw {
@@ -1853,6 +1866,9 @@ declare module 'leaflet' {
     interface DrawModeOptions extends SnappingOptions {
       /** Require the last point of a shape to be snapped. (default: false). */
       requireSnapToFinish?: boolean;
+
+      /** Enable finishing drawing shapes (Line, Polygon, Cut) by pressing the Enter key when enough vertices are placed. (default: false) */
+      finishOnEnter?: boolean;
 
       /** Show helpful tooltips for your user (default:true). */
       tooltips?: boolean;

